# EMO-BON glossary 

## Physical concepts
| **Term** | **Definition** | |
|----------|-----------------|------|
| **Partner** | An EMO BON member, which is typically but not exclusively an institute. | 
| **Station** | An EMO BON Station. Stations collect EMO BON samples. Stations may have multiple observatories, and each observatory can involve contributions from one or more partners. Stations are recognised by some local code assigned by the partner and have a geolocation.|
| **Observatory** | An EMO BON organisational unit linked to the collection of a specific sample type (e.g. water column, soft sediment) from a fixed, pre-determined location. While _technically_ an observatory is tied to a sample type, this distinction is often ignored in casual use since the observatory's base name (obs_id) is the same for all sample types.  | Note: definition may need an update once the ARMS units are fully incorporated in EMO BON.  
| **Sampling event** | A sampling action performed at a particular observatory at a specific time, resulting in the collection of one or more samples. Typically has some other measurements of the environment assigned to it. |
| **Material Sample** | Refers to a material sample collected during a sampling event. Each unique material sample has a unique material sample ID. Also used to refer to the sample material that was sequenced, where the physical sample no longer exists but it is virtually present via its sample ID.|

## Digital concepts
| **Term** | **Definition** | |
|----------|-----------------|-------|
| **Logsheet** | The spreadsheets in which the observatories write their sample and event data. The source spreadsheets are on the EMO BON googledrive. from where they are harvested as CSV into EMOBON's GH space. The "transformed" logsheets are those that have been then subjected to a date-range selection and a QC. ||
| **EMO BON data** | Here we mean: the content of the logsheets, which are filled by the observatories to describe their collected samples; the sequences in ENA;the outputs from bioinformatics processing. | And once ARMS units are incorporated, this will also include ARMS images.
| **EMO BON metadata** | Here we mean the data that is used specifically to describe EMO BON data, performing the function of allowing discovery, understanding, organising, cataloguing, etc. Metadata are recorded in the rocrate-metadata.json files; they are added to ENA accessions; they are in files in the EMO BON repos governance-data, sequencing-data, observatory-profile, among others. |
| **EMO BON record** | A digital representation of a sampling event, capturing the relevant data and metadata associated with it. There is no fixed idea of what is included in an EMO BON record, as that depends on the system that these records are being held in; for example, EMO BON records in EurOBIS and in Blue Cloud will not necessarily be the same. |
| **Catalogue asset** | The unit of registration into a datasets metadata catalogue, i.e. a specific "EMO BON record" in a specific catalogue. Can be a single data file or a set of files.|
| **EMO BON Repository** | A GitHub repository that contains EMO BON data and metadata. <br>A GitHub repository represents a storage location for files and their version histroy, managed using Git version control which allows users to track changes, collaborate with others, and maintain a complete record of the project's development over time. |
| **EMO BON RO-Crate** | EMO BON RO-Crates contain data associated with either: logsheets from observatories, MetaGOflow runs, sequencing metadata. Usually our RO-Crates map one-to-one to single repositories, but some (e.g. for the analysis results) RO-Crates do get grouped into one repository. A RO-Crate is manifest via a RO-Crate-metadata.json file  <br>A RO-crate is a collection of data files, metadata, and contextual information that organises research data in a structured format, enabling uniquely identifying and referencing, easy sharing, reuse, and understanding in both machine-readable and human-readable forms. They are built according to the RO-Crate specification managed at [https://www.researchobject.org/ro-crate/](https://www.researchobject.org/ro-crate/)|
| **EM BON ro-crate-metadata.json** | A `ro-crate-metadata.json` file describes the contents of an EMO BON RO-Crate. <br>A `ro-crate-metadata.json` file is a JSON-LD file that provides a detailed description of the contents and structure of an RO-Crate. It maps relationships between files and their metadata, ensuring traceability, context, and purpose for data within research workflows.|
| **sequence** | A DNA string. Specifically, we mean (raw) sequences as produced from the material samples by Genoscope and held on their cloud drive and then archived on ENA. |
| **processed sequences / OTUs / ASVs** | These are sequences that have been processed by a bioinformatics code to a stage where they can be/have been compared to taxonomic reference libraries. |
| **DVC Stub File** | Inside our datamanagement solution we make use of [dvc.org](dvc.org) to connect our "EMO BON Repositories" (git based) to external large file storage locations.  This is to overcome practical limits with large files in combintation with git. The usage of dvc.org manifests itself through our repositories containining so called "dvc stub files".  These have a `.dvc` file extension and are local stand-in for the original large files.  They contain the neccessary pointers for the dvc.org software to pull in the actual large files upon request. ||
